Anytype/Sources/PresentationLayer/Modules/Membership/Models/MembershipTier.swift

```diff
@@ -1,7 +1,92 @@
-enum MembershipTier: String, Identifiable {
+enum MembershipTier: String, Identifiable, CaseIterable {
     case explorer
     case builder
     case coCreator
     
     var id: String { rawValue }
 }
+
+import SwiftUI
```

Comment:
import in the middle looks interesting ))

Reply:
+